src/pages/dashboard/nodeSignIn/NodeSignInPTForm.tsx

```diff
@@ -142,7 +132,7 @@ const NodeSignInPTForm = ({ goBack, signInData }: Props) => {
   };
 
   const disabledSubmit = () =>
-    isPartnerPSPChecked === isPartnerPSP && isPartnerCIChecked === isPartnerCI;
+    isPartnerPSPChecked === orgInfo.types.isPspBroker && isPartnerCIChecked === orgInfo.types.isEcBroker;
```

Comment:
Same comment, `orgInfo.types.isPspBroker / orgIsPspDirect` and `orgInfo.types.isEcBroker / orgIsEcDirect` ?